Servlet/src/main/java/top/jsoft/service/GetService.java

```diff
@@ -0,0 +1,55 @@
+package top.jsoft.service;
+
+import top.jsoft.dao.OrderDAO;
+import top.jsoft.model.Order;
+import top.jsoft.model.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+public class GetService {
```

Comment:
Please rename to OrderService

Reply:
Done, i understand that it is better to create your own service for each servlet.